### DISCUSSION TITLE
Any tutorials using ink4 (with ESM)?  What about create-ink-app?

### DISCUSSION THREAD

**Discussion by artemis-prime** [vadimdemedes/ink#574] [Q&A]
Create create-ink-app outputs commonjs.  Is there any plan to update it?

Also, are there any tutorials or materials specifically using ink 4 features and syntax?  (other than the examples in this repo)

best, 
Artemis

**Comment by vadimdemedes** [comment]
`create-ink-app` just got updated to support Ink 4. Update to the latest version and you should be good to go.

> Also, are there any tutorials or materials specifically using ink 4 features and syntax? (other than the examples in this repo)

Not that I'm aware of since Ink 4 got released not that long ago and it's a pretty niche tool. Readme and examples in this repository are the most up-to-date resources to build with Ink.
